lib/truffle/java/env_java.rb

```diff
@@ -37,4 +37,10 @@ def inspect
   end
 end
 
-ENV_JAVA = Java::SystemPropertyMap.new
+verbose = $VERBOSE
+begin
+  $VERBOSE = nil
+  ENV_JAVA = Java::SystemPropertyMap.new
+ensure
+  $VERBOSE = verbose
+end
```

Comment:
Why do we need ENV_JAVA defined before this? There are some gems using it but not requiring `java`?
Maybe one way to fix this would be to define ENV_JAVA in core, and just use a couple primitives.

Another way to avoid the warning would be to `remove_const :ENV_JAVA`.
But this seems fine for now.